Werkzeug 1.0.* fails Safety check

Recently, [Werkzeug](https://github.com/pallets/werkzeug) 1.* has been flagged for vulnerabilities by [Safety](https://github.com/pyupio/safety). This is causing our GitHub Actions workflows to fail:
```
+==============================================================================+
|                                                                              |
|                               /$$$$$$            /$$                         |
|                              /$$__  $$          | $$                         |
|           /$$$$$$$  /$$$$$$ | $$  \__//$$$$$$  /$$$$$$   /$$   /$$           |
|          /$$_____/ |____  $$| $$$$   /$$__  $$|_  $$_/  | $$  | $$           |
|         |  $$$$$$   /$$$$$$$| $$_/  | $$$$$$$$  | $$    | $$  | $$           |
|          \____  $$ /$$__  $$| $$    | $$_____/  | $$ /$$| $$  | $$           |
|          /$$$$$$$/|  $$$$$$$| $$    |  $$$$$$$  |  $$$$/|  $$$$$$$           |
|         |_______/  \_______/|__/     \_______/   \___/   \____  $$           |
|                                                          /$$  | $$           |
|                                                         |  $$$$$$/           |
|  by pyup.io                                              \______/            |
|                                                                              |
+==============================================================================+
| REPORT                                                                       |
| checked 85 packages, using free DB (updated once a month)                    |
+============================+===========+==========================+==========+
| package                    | installed | affected                 | ID       |
+============================+===========+==========================+==========+
| werkzeug                   | 1.0.1     | <2.0.2                   | 42050    |
+==============================================================================+
Exception ignored in: <_io.TextIOWrapper name='<stdout>' mode='w' encoding='UTF-8'>
BrokenPipeError: [Errno 32] Broken pipe
Error: Process completed with exit code 255.
```

The vulnerability is as follows:
```json
        {
            "advisory": "Werkzeug version 2.0.2 improves the security of the debugger cookies. \"SameSite\" attribute is set to \"Strict\" instead of \"None\", and the secure flag is added when on HTTPS.",
            "cve": "PVE-2021-42050",
            "id": "pyup.io-42050",
            "specs": [
                "<2.0.2"
            ],
            "v": "<2.0.2"
        },
```

We need to either migrate to Flask/Werkzeug 2.0 (a substantial project), or add an exception for this in our pipeline.

--- a/.github/workflows/ci.yaml
+++ b/.github/workflows/ci.yaml
@@ -29,7 +29,7 @@ jobs:
     - name: Check for vulnerabilities in libraries
       run: |
         pip install safety
-        pip freeze | safety check
+        pip freeze | safety check -i 42050
     - name: Lint with Flake8
       run: |
         flake8

--- a/Dockerfile
+++ b/Dockerfile
@@ -12,7 +12,7 @@ COPY requirements.txt requirements.txt
 RUN pip install -e .
 RUN pip install gunicorn
 RUN pip install safety && \
-    pip freeze | safety check
+    pip freeze | safety check -i 42050
 
 COPY *.py docker-dev.conf boot.sh ./
 COPY servicex/ ./servicex

